std/parallelism.d

```diff
@@ -4914,3 +4914,10 @@ version (parallelismStressTest)
     try foreach (rnd; rndGen.parallel) break;
     catch (ParallelForeachError e) {}
 }
+
+nothrow pure @safe unittest
+{
+    // Parallelism (was broken by inferred return type "immutable int")
+    import std.algorithm.comparison;
+    auto t = task!cmp("foo", "bar");
+}
```

Comment:
Moving this test seems unrelated

Reply:
This has to do with nice circular dependency problem - if std.regex.ir has module ctor then during unittest compilation a cycle between std.algorithm.comparison and std.parallelism is hit. I'd wish we didn't consider unittests to be part of the module and have them compiled somehow separately but it is what it is. The simplest path out of this hell was to move test to more heavily dependent module - std.parallelism.